### PR TITLE
Use geth personal.signTransaction

### PR DESCRIPTION
**Pull request by ryanio**
This code changes the sendTransaction window in Mist to use geth to first sign the transaction before broadcasting the signed tx to the network.

This allows us to broadcast the transaction however we wish in the future, like an override to send it through a remote node (infura).

To test, try sending a transaction in mist.

### CODE REVIEW

File: interface/client/templates/popupWindows/sendTransactionConfirmation.js [hunk 1]
```diff
@@ -398,58 +398,94 @@ Template['popupWindows_sendTransactionConfirmation'].events({
...
+      pw = null;
+      TemplateVar.set('unlocking', false);
+      if (error) {
         Tracker.afterFlush(function() {
           template.find('input[type="password"]').value = '';
           template.$('input[type="password"]').focus();
         });
-        if (e.message.indexOf('Unable to connect to socket: timeout') !== -1) {
+        switch(error) {
+          case
```
Comment (marcgarreau) [inline interface/client/templates/popupWindows/sendTransactionConfirmation.js]: syntax 💣 here

**Comment by ryanio** [comment]
thanks good catch, on hold while we update to beta.33 which has the method

**Comment by marcgarreau** [comment]
Nearly synced and getting spammed with: `TypeError: Cannot read property 'peerCount' of undefined`

**Comment by ryanio** [comment]
Merging into #3776

**Pull request closed without merging**